index.html

```diff
@@ -3,9 +3,34 @@
   <head>
     <meta charset="utf-8">
     <title>super-fantastic-dashboard</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
```

Comment:
Let's move all CDN scripts towards the bottom of the page so that it's non-blocking.